docs/install/sandboxes.rst

```diff
@@ -234,6 +234,87 @@ Notice that we can get the number of members of upstream clusters, number of req
 fulfilled by them, information about http ingress, and a plethora of other useful
 stats.
 
+Zipkin Tracing
```

Comment:
nit: This page is absolutely huge. Can we potentially break this into multiple pages, one for each example? You didn't cause this problem but would be great to fix now.